src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -18,7 +18,7 @@
 
 // Visualize styles
 @import './visualize/index';
-@import './visualize_embeddable/index';
+@import '../../visualizations/public/embeddable/index';
```

Comment:
Sorry I didn't mean you should change the import path in this file. You shouldn't be importing SASS files from other plugins. Each plugin needs their own manifest file `index.scss`. You can follow the pattern setup in this folder https://github.com/elastic/kibana/tree/c5306961d3ac4826ac044dc8d4b9031d8b0f4a8b/src/legacy/core_plugins/newsfeed/public